testing/src/main/java/org/oppia/testing/TestCoroutineDispatchers.kt

```diff
@@ -23,18 +17,41 @@ import javax.inject.Inject
  * mode so that clock coordination is consistent between Robolectric's scheduler and this utility
  * class, otherwise unexpected inconsistencies may arise.
  *
- * *NOTE TO DEVELOPERS*: This class is NOT yet ready for broad use until after #89 is resolved.
- * Please ask in oppia-android-dev if you have a use case that you think requires this class.
- * Specific cases will be allowed to integrate with if other options are infeasible. Other tests
- * should rely on existing mechanisms until this utility is ready for broad use.
+ * It's also recommended that all Espresso tests register the idling resource provided by this class
+ * (see [registerIdlingResource]) to get the same synchronization benefits of the
+ * Robolectric-specific API methods (e.g. [runCurrent], [advanceTimeBy], and [advanceUntilIdle]).
+ *
+ * *NOTE TO DEVELOPERS*: This class is NOT yet ready for broad use until after #89 is fully
+ * resolved. Please ask in oppia-android-dev if you have a use case that you think requires this
```

Comment:
Is this still true?

Reply:
Eh, I removed it in #1624, so sort of? It's true until removed in that PR. :)